nethttp/server.go

```diff
@@ -68,12 +59,11 @@ func Middleware(tr opentracing.Tracer, h http.Handler, options ...MWOption) http
 		ext.HTTPMethod.Set(sp, r.Method)
 		ext.HTTPUrl.Set(sp, r.URL.String())
 		ext.Component.Set(sp, "net/http")
-		w = &statusCodeTracker{w, 200}
 		r = r.WithContext(opentracing.ContextWithSpan(r.Context(), sp))
 
-		h.ServeHTTP(w, r)
+		m := httpsnoop.CaptureMetrics(h, w, r)
```

Comment:
It's not just time overhead, but also memory allocations. The CaptureMetrics is a lot heavier than what's needed here if we only want to capture the status code